Run3Detector/analysis/utilities/milliqanCuts.py

```diff
@@ -15,6 +39,17 @@ def combineCuts(self, name, cuts):
             else:
                 self.events[name] = self.events[cut]
 
+    # Dummy cut for use while construcing the cutflow mechanics
+    def neverCut(self, cutName=None, cut=False):
+        if cut: self.events = self.events
+        self.cutflowCounter()
+
+    # Dummy cut for use while construcing the cutflow mechanics
+    def idCut(self, cutName=None, cut=False):
```

Comment:
if this is no longer needed delete

Reply:
Removed, but I'd like to keep the ```neverCut``` for now, just as an extra debugging tool